README.md

```diff
@@ -1,4 +1,6 @@
 # (in private beta) playit agent
 
-This is beta software for connecting to the services provided by [https://new.playit.gg](https://new.playit.gg). 
-The APIs and protocols used are likely to change multiple times before leaving the beta. 
\ No newline at end of file
+This is beta software for connecting to the services provided by [https://new.playit.gg](https://new.playit.gg).
+The APIs and protocols used are likely to change multiple times before leaving the beta.
+
+You will need Perl and Visual Studio to compile OpenSSL
```

Comment:
You should mention what platform this is a requirement for. I'm guessing Windows.